.gitignore

```diff
@@ -2,4 +2,5 @@
 .meteor/local
 .meteor/meteorite
 private/local-settings.json
-client/style.sass
```

Comment:
Yeah, seems odd.  Maybe because I added .idea for Webstorm to the end so the new line was swapped?